packages/dds/tree/src/test/feature-libraries/chunked-forest/codec/chunkDecoding.spec.ts

```diff
@@ -280,7 +280,12 @@ describe("chunkDecoding", () => {
 
```

Comment:
You added more options and code to chunkDecoding.ts. THose should be tested:

At least one test which uses IncrementalChunkDecoder, and at least one which passes in incrementalDecoder to the DecoderContext (those can be combined) would be good to provide basic unit test coverage for the new code.

Reply:
Added tests for these.